<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Panel</class>
 <widget class="QWidget" name="Panel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>812</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="Iniciar">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>30</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Iniciar</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_nombre1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Snap ITC</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_nombre2">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>20</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Snap ITC</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGraphicsView" name="view">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>771</width>
     <height>361</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_vida1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Snap ITC</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_vida2">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>60</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Snap ITC</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
